test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -15,7 +15,7 @@ namespace DynamoCoreWpfTests
 {
     public class PythonNodeCustomizationTests : DynamoTestUIBase
     {
-        private readonly List<string> expectedEngineMenuItems = Enum.GetNames(typeof(PythonNodeModels.PythonEngineVersion)).ToList();
+        private readonly List<string> expectedEngineMenuItems = Enum.GetNames(typeof(PythonNodeModels.PythonEngineVersion)).Skip(1).ToList();
```

Comment:
? comment on this maybe?

Reply:
I think I'd rather change the code to make it more clear.